contrib/conan/configs/install.ps1

```diff
@@ -0,0 +1,53 @@
+# Copyright (c) 2020 The Orbit Authors. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+
+$conan = Get-Command conan -ErrorAction Stop
+
+function conan_disable_public_remotes() {
+  $remotes = "bintray", "conan-center", "bincrafters"
+  foreach ($remote in $remotes) {
+    $process = Start-Process $conan.Path -Wait -NoNewWindow -ErrorAction Stop -PassThru -ArgumentList "remote","disable",$remote
+    if ($process.ExitCode -ne 0) { Throw "Error while disabling conan-remote $remote." }
+  }
+}
+
+$process = Start-Process $conan.Path -Wait -NoNewWindow -ErrorAction Stop -PassThru -ArgumentList "config","install","$PSScriptRoot\windows"
```

Comment:
Myunderstanding is that this line actives the public remotes. Where is this configured? Maybe add a comment.

Reply:
Comment added. +1